pgcli/pgexecute.py

```diff
@@ -215,6 +221,9 @@ def connect(self, database=None, user=None, password=None, host=None,
         self.host = host
         self.port = port
 
+        if not self.host:
+            self.host = self.socket_directory()
```

Comment:
I didn't know that postgresql does that. Can you verify that you'd be able to connect with psycopg2 if you provide `host=/tmp`, and that reconnect behavior doesn't suffer when you run pgcli without providing hostname? If yes, we can keep that behavior.